tests/functional/notification_test.php

```diff
@@ -82,6 +82,6 @@ public function test_mark_notifications_read()
 		// Get form token
 		$link = $crawler->selectLink($this->lang('NOTIFICATIONS_MARK_ALL_READ'))->link()->getUri();
 		$crawler = self::request('GET', substr($link, strpos($link, 'ucp.')));
-		$this->assertCount(0, $crawler->filter('#notification_list_button strong'));
+		$this->assertEquals(0, $crawler->filter('#notification_list_button strong.badge.hidden')->text());
```

Comment:
But then you would not be checking for the actual contents, just the existence of the element with the class, right?
What I wanted to achieve is testing both the right element and class, and the actual content (0).
What os the right way to do that?